CHANGELOG.adoc

```diff
@@ -4,7 +4,18 @@ This file documents all notable changes to https://github.com/devonfw/sonar-devo
 
 == 2020.04.002
 
+* https://github.com/devonfw/sonar-devon4j-plugin/issues/73[#73]: Increased code coverage of tests
 * https://github.com/devonfw/sonar-devon4j-plugin/issues/77[#77]: Fixed a bug where the architecture.json could not be found in multi-module projects
+* https://github.com/devonfw/sonar-devon4j-plugin/issues/81[#81]: Improved redundant code parts
+* https://github.com/devonfw/sonar-devon4j-plugin/issues/83[#83]: Integrated the devonfw maven-parent as parent pom.xml
+* https://github.com/devonfw/sonar-devon4j-plugin/issues/88[#88]: Fixed remaining code smells thrown by SonarCloud analysis
+
+Further, there were some documentation improvements with the following PRs:
```

Comment:
So far we do not distinguish between issues and PRs here nor do we distinguish between documentation and code changes (that also often go together). Instead we try the following:

* provide an initial sentence to sommarize the release at the top. E.g. "This release fixes some bugs and adds some minor improvements:"
* if we have both an issue and a PR for a story then we link the issue here. Further both issue and PR shall have the milestone assigned and shall be linked to each other (at the side-bar in `Linked pull requests` for the issue and `Linked issues` for the PR) and obviously both shall be closed.

IMHO your approach would also be fine, but we should avoid changing the old history of existing CHANGELOG files and should try to keep them homogenous and consistent within devonfw. Therefore I suggest that we follow the conventions we have set by `devon4j`, `devonfw-ide`, etc.

Reply:
True, I will add a summarizing sentence at the start. Should I also just put the two documentation PRs in the same list as the issues?
To your second point: The links are all issues which are assigned to the milestone they are written under.

Reply:
And to #44: Do you remember with which PR it was fixed? I would add it to the release that we were working towards at that time